### PR TITLE
Delegate batch signatures for DIP-145

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======

Implementation layer should have the details about validation instead of signatures on announcements

See DIP-#145 for details

Closes: #145 

Solution
========
Reduce the validation cost and increase transparency by removing the signature from the individual Announcements and requiring that the implementation have a way to validate. EVM Implementation being that the batch submission is from a user or delegate directly

Change summary:
---------------
- [x] Updated Spec Versions
- [x] Remove signatures from Announcements
- [x] Remove Signatures.md file
- [x] Remove createdAt from Announcements as it was only there for replay protection
- [x] Remove other references to signatures for duplicate handling that was required for replay attacks.

Steps to Verify:
----------------
1. No more signatures on any Announcement Types
2. No more createdAt on any Announcement Types
3. Version bump for Ethereum implementation

### CODE REVIEW

File: pages/DSNP/Types/GraphChange.md [hunk 1]
```diff
@@ -32,48 +30,24 @@ Different change types have different meanings.
...
-### createdAt
-
-- MUST be set to the milliseconds since Unix epoch at time of signing
-- MUST be unique for the given `fromId` and `objectId` pair
-
 ### fromId
 
 - MUST be a [DSNP User Id](../Identifiers.md#dsnp-user-id)
-- MUST be the [signer](../Signatures.md) of the Announcement
+- MUST directly or via a chain of delegation have authorized the creation of the Announcement
```
Comment (saraswatpuneet) [inline pages/DSNP/Types/GraphChange.md:37]: How about 

MUST have, directly or via a chain of delegation, authorized the creation of the Announcement

**Review by saraswatpuneet — APPROVED** [review]
Looks good just had one comment about readability of that MUST

File: pages/DSNP/Types/GraphChange.md [hunk 2]
```diff
@@ -32,48 +30,24 @@ Different change types have different meanings.
...
 
-### signature
-
-- MUST be an [Announcement Signature](../Signatures.md) over all fields except the signature field
-
 ## Non-Normative
 
-### Replay Attacks
-
-Clients must ignore any Graph Change event that comes after another event with the same signature.
```
Comment (shannonwells) [inline pages/DSNP/Types/GraphChange.md:57]: I don't see how this is related to DIP-145.

**Review by shannonwells — COMMENTED** [review]
I'm not sure what the created_at field has to do with removing signature validation from announcements.

File: pages/DSNP/Types/GraphChange.md (hunk 2 above)
Comment (wilwade) [reply pages/DSNP/Types/GraphChange.md:57]: @shannonwells it is unclear. Here's why I did it in this DIP:

1. Replay attacks are only possible with announcement level signatures.
2. created_at was only added for replay attack prevention.

So once we take away signatures, neither of these have a reason to exist either.

File: pages/DSNP/Types/GraphChange.md (hunk 1 above)
Comment (wilwade) [reply pages/DSNP/Types/GraphChange.md:37]: Good idea. I'll update.

Comment (wilwade) [reply pages/DSNP/Types/GraphChange.md:37]: @saraswatpuneet Updated to `MUST have authorized the creation of the Announcement, either directly or via a transparent chain of delegation`. Does that work?

Comment (saraswatpuneet) [reply pages/DSNP/Types/GraphChange.md:37]: @wilwade  yes that looks great thank you

**Review by enddynayn — APPROVED** [review]

File: pages/DSNP/Announcements.md [hunk 3]
```diff
@@ -42,6 +42,15 @@ For example, if a user creates a Reaction Announcement, they may remove that rea
 
 ## Non-Normative
 
+### Duplicate Announcements
+
+Due to the distributed nature of DSNP, duplicate Announcements are possible from time to time.
```
Comment (aramikm) [inline pages/DSNP/Announcements.md:47]: Do we need to talk about how an end service can dedup these announcements or that's just not a protocol concern?

**Pull request merged**